tasks.php

```diff
@@ -788,12 +804,12 @@ function handle_action_on_a_specified_task()
     elseif ($action == 'notify_me') {
         $userSettings =& Settings::get_Settings($pguser);
         $userSettings->add_value('taskctr_notice', $task_id);
-        TaskDetails($task_id);
+        metarefresh(0, "$tasks_url?action=show&task_id=$task_id");
```

Comment:
I'm uncertain this will work because things have already been output so a header redirect will fail. We either need to continue with the old method or move this up and out of this function and do the evaluation earlier before TaskHeader().